wiki/Help_Centre/Client/id.md

```diff
@@ -0,0 +1,518 @@
+
+**Menginstal ulang osu! akan menghapus semua skor lokal yang kamu miliki.**
+
+Skor-skor ini sayangnya tidak dapat dikembalikan.
+
+Meskipun demikian, skor-skor yang kamu torehkan ketika kamu bermain secara *online* tidak akan hilang - kamu hanya perlu mengunduh ulang beatmap-beatmap yang pernah kamu mainkan untuk kembali mendapatkan skor-skor ini.
+
+Apabila kamu memiliki osu!supporter, kamu dapat menggunakan filter `Pernah Dimainkan` pada opsi pencarian `Riwayat Permainan` yang ada pada [laman daftar beatmap](https://osu.ppy.sh/beatmapsets?played=played&s=ranked) untuk mengetahui beatmap-beatmap apa saja yang pernah kamu mainkan sebelumnya.
+
+#### Apakah kamu sedang berada pada mode permainan yang kamu kehendaki?
+
+**osu! memiliki papan peringkat lokal yang terpisah untuk keempat mode permainan yang ada (osu!, osu!taiko, osu!catch, dan osu!mania).**
+
+Masalah ini umum dijumpai oleh para pemain yang tidak sengaja memainkan beatmap di mode permainan lain. Apabila kamu baru saja memainkan beatmap di mode permainan lain, kamu harus terlebih dahulu kembali ke mode permainan utamamu untuk dapat memunculkan papan peringkat lokal yang ada sebelumnya.
+
+Kamu dapat mengubah mode permainan yang kamu kehendaki melalui tombol `Mode` yang ada pada pojok kiri bawah layar Song Select.
+
+#### Apakah kamu baru saja selesai mengunduh sekian banyak beatmap (melalui bundel/paket beatmap atau lain sebagainya) dalam satu waktu?
+
+**Dalam kasus demikian, osu! membutuhkan waktu untuk dapat memproses skor-skor yang ada.**
+
+Kamu hanya perlu bersabar sembari menunggu osu! memuat skor-skor yang pernah kamu torehkan sebelumnya.
+
+### Apa gerangan jejak berwarna putih yang mengekori gerakan kursor saya ini?
+
+**Jejak putih tersebut merupakan fitur *smoke* yang dapat kamu gunakan untuk menggambar hal apapun yang kamu inginkan. Para pemain yang sedang menontonmu juga akan dapat melihat jejak putih ini.**
+
+Secara *default*, fitur *smoke* ini akan muncul setiap kali kamu menahan tombol `C` pada *keyboard*. Kamu dapat mengubah tombol ini melalui opsi `Konfigurasi Tombol` yang ada pada menu `Options`, tepatnya pada sub-bagian `osu!`.
+
+### Bagaimana caranya agar saya dapat menyimpan tayangan ulang (*replay*) dari skor yang baru saja saya torehkan?
+
+**Kamu dapat menyimpan tayangan ulang yang kamu kehendaki dengan menekan tekan tombol `F2` pada layar skor.**
+
+Tayangan-tayangan ulang yang tersimpan dapat kamu akses pada folder `/Replays/` yang ada pada direktori osu! milikmu.
+
+Lebih jauh, sistem internal osu! juga menyimpan berbagai tayangan ulang pada folder `Data/r/` yang tersembunyi di dalam direktori osu! yang ada.
+
+Harap dicatat bahwa apabila karena suatu alasan tayangan ulang milikmu hilang atau tidak lagi tersedia secara *online*, tayangan ulang tersebut akan hilang selama-lamanya.
+
+### osu! memberitahu saya bahwa versi osu! yang saya gunakan sudah terlalu lawas (*my game client is too old*)!
```

Comment:
is the phrase intentionally left in English? if you are referring to the in-game label, I vaguely remember it saying ***Your** game client is too old!*

Reply:
oh yeah this line is intentionally left like that for convenience as (to my knowledge) there's no Indonesian equivalent of this particular error string 

...but yeah this should be *your* instead of *my* indeed, my bad :P